tests/pack_test.cpp

```diff
@@ -100,8 +98,9 @@ void end() {
 
 void packTest() {
 
-  printf("Sending fields to GPU...\n"); fflush(stdout);
-  
+  printfQuda("Sending fields to GPU...\n");
+  fflush(stdout);
```

Comment:
I know that has been here for a long time but so we need the flush?

Reply:
I don't know. I strongly suspect a `fflush` call is not needed.

Reply:
Fixed